proto/resource_ops.proto

```diff
@@ -0,0 +1,80 @@
+syntax = "proto3";
+
+package rs.tokio.console.resource_ops;
+
+import "google/protobuf/timestamp/timestamp.proto";
+import "google/protobuf/duration.proto";
+import "common.proto";
+
+
+// An AsyncOp state update.
+message ResourceOpUpdate {
+    repeated ResourceOp new_resource_ops = 1;
+}
+
+message ResourceOp {
+    // The resources's ID.
+    //
+    // This uniquely identifies this op across all *currently live*
+    // ones.
+    common.MetaId id = 1;
```

Comment:
I don't think I fully understand the difference of these 3 IDs.

Reply:
I should have clarified this more. 

- `common.MetaId metadata`  is the id of the metadata for tracing event that describes this op
- `common.MetaId id ` should really be the unique identifier for this resource op. Since this resource op is represented by an event in tracing, not a span it does not have a span id. This is why I have used the metadata id for the purpose as well. Not sure whether this is correct as I am not sure this will actually be unique. Namely, there could be two invocations of a resource op that have the exact metadata and will produce the very same metadata id. Need to revisit that. Also I am not too sure we actually need any unique identifier for these. Suggestions welcome.
- `common.SpanId resource_id` is the id of the resource. Any resource operation should be attached to a resource. Like if you call `reset` on a timer, this will produce a ResourceOp and the resource id will be the id of the  main resource span.

Reply:
>   * `common.MetaId id ` should really be the unique identifier for this resource op. Since this resource op is represented by an event in tracing, not a span it does not have a span id. This is why I have used the metadata id for the purpose as well. Not sure whether this is correct as I am not sure this will actually be unique. Namely, there could be two invocations of a resource op that have the exact metadata and will produce the very same metadata id. Need to revisit that. Also I am not too sure we actually need any unique identifier for these. Suggestions welcome.

This will not be unique; it will be shared across all `ResourceOp` events originating from the same `tracing::event!` macro. In fact, `ResourceOp.id` will _always_ be equal to `ResourceOp.metadata`.

I think this field should probably just be removed entirely. Since these resource ops correspond to single events, and we won't emit more data about a specific resource op more than once, I don't think they need a unique identifier. If we do want them to have unique identifiers, we should probably just generate them with a global counter in the subscriber.